### ISSUE TITLE
BUG: az webapp up | 'NoneType' object has no attribute 'upper'

### ISSUE THREAD

**Issue by AllanFerencz** [Azure/azure-cli#11232]
## Describe the bug

Following the tutorial "Host a web application with Azure App service" unit 6 Exercise causes an error when using "az webapp up" to deploy the "dotnet new mvc" blank site.

[https://docs.microsoft.com/en-us/learn/modules/host-a-web-app-with-azure-app-service/](https://docs.microsoft.com/en-us/learn/modules/host-a-web-app-with-azure-app-service/)

**Command Name**
`az webapp up`

**Errors:**
```
'NoneType' object has no attribute 'upper'
Traceback (most recent call last):
python3.6/site-packages/knack/cli.py, ln 206, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 603, in execute
    raise ex
cli/core/commands/__init__.py, ln 661, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
cli/core/commands/__init__.py, ln 652, in _run_job
    cmd_copy.exception_handler(ex)
...
cli/command_modules/appservice/custom.py, ln 2993, in webapp_up
    create_app_service_plan(cmd, rg_name, plan, _is_linux, False, sku, 1 if _is_linux else None, location)
cli/command_modules/appservice/custom.py, ln 1390, in create_app_service_plan
    sku = _normalize_sku(sku)
cli/command_modules/appservice/utils.py, ln 20, in _normalize_sku
    sku = sku.upper()
AttributeError: 'NoneType' object has no attribute 'upper'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

1. Create a Sand box account.
2. Open Bash in Azure Cloud Shell
3. Run the below commands

```
dotnet new mvc --name BestBikeApp
```

```
APPNAME=$(az webapp list --query [0].name --output tsv)
APPRG=$(az webapp list --query [0].resourceGroup --output tsv)
APPPLAN=$(az appservice plan list --query [0].name --output tsv)
APPSKU=$(az appservice plan list --query [0].sku.name --output tsv)
APPLOCATION=$(az appservice plan list --query [0].location --output tsv)
```

```
az webapp up --name $APPNAME --resource-group $APPRG --plan $APPPLAN --sku $APPSKU --location "$APPLOCATION"
```

**Specific command that errors**
- `az webapp up --name {} --resource-group {} --plan {} --sku {} --location {}`

## Expected Behavior
No Errors / Deployed site

## Environment Summary
```
Linux-4.15.0-1063-azure-x86_64-with-debian-stretch-sid
Python 3.6.5
Shell: bash

azure-cli 2.0.76

```

**Comment by fengzhou-msft** [comment]
This is fixed by #11019

**Comment by roryhanlon** [comment]
Why is this bug marked closed? Its still happening exactly as the OP described!
